README.md

```diff
@@ -1,88 +1,129 @@
+### AWS Mobile Client
 
-``` gradle
-buildscript {
-    repositories {
-        mavenLocal() // This should ideally appear before other repositories
-    }
-    dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.1'
+The default plugins for Amplify Android rely on the AWS Mobile Client to
+provide authentication with AWS services.
+
+Please see [Getting Started with Authentication](https://docs.amplify.aws/lib/auth/getting-started?platform=android)
+for full details.
+
+In summary, you need to ensure `AWSMobileClient` is [initialized](https://docs.amplify.aws/lib/auth/getting-started/q/platform/android#initialization)
+before interfacting with `Amplify`:
+
+```
+AWSMobileClient awsAuth = AWSMobileClient.getInstance();
+Context context = getApplicationContext();
+awsAuth.initialize(context, new Callback<UserStateDetails>() {
+    @Override
+    public void onResult(UserStateDetails userStateDetails) {
+        Amplify.addPlugin(new AWSApiPlugin()); // For example
+        Amplify.configuration(context);
+        Toast.makeText(context, "OK!", Toast.LENGTH_SHORT);
     }
-}
 
-allprojects {
-    repositories {
-        mavenLocal() // This should ideally appear before other repositories
+    @Override
+    public void onError(Exception error) {
+        Toast.makeText(context, "Uh oh...", Toast.LENGTH_SHORT);
     }
-}
+});
 ```
 
-## Using Amplify
-
-### Java 8 Compatibility
+Please note that `AWSMobileClient` initialization is ___not___ required
+when using the `AWSApiPlugin` or `AWSDataStorePlugin` with _api key_ as the
+authorization mode.
 
-Amplify Android uses Java 8 features. Please add compile options inside app's `build.gradle` like following:
+## License
 
-``` gradle
-android {
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
-    }
-}
-```
+This library is licensed under the [Apache 2.0 License](./LICENSE).
 
-### Using AWS Mobile Client
+## Report a Bug
 
-Amplify relies on the AWS Mobile Client for authentication with AWS services. Please refer to the [this documentation](https://docs.amplify.aws/lib/auth/getting-started?platform=android) for details.
+We appreciate your feedback -- comments, questions, and bug reports. Please
+[submit a GitHub issue](https://github.com/aws-amplify/amplify-android/issues),
+and we'll get back to you.
 
-You need to ensure `AWSMobileClient` is [initialized](https://docs.amplify.aws/lib/auth/getting-started?platform=android#initialization) before you initalize and configure `Amplify`. Please note that AWSMobileClient initialization is not required when using API or DataStore category with _API Key_ as the authorization mode.
+## Contribute to the Project
 
-## License
+Please see the [Contributor's Guide](./CONTRIBUTING.md).
 
-This library is licensed under the Apache 2.0 License. 
+--------------------------------
 
+[amplify.aws](https://amplify.aws)
```

Comment:
Idle thought: This link just floating here at the bottom might not be necessary. I just changed the repo's website to this URL, which I think is more visible.